.github/workflows/ci.yml

```diff
@@ -72,7 +72,12 @@ jobs:
         uses: microsoft/setup-msbuild@v1.3
       - name: Build
         run: msbuild src -p:Configuration=Release -restore -m -verbosity:minimal
-      - name: Upload packages
+      - name: Zip PowerShell module
+        if: matrix.transport == 'Msmq' && matrix.storage == 'RavenDB35'
```

Comment:
Should we change this to use a condition that is more likely to still exist in ~3 years? (Line 81 too)

Another option is to add a `upload-packages: true` to one of the configs but the addition of the storage aspect of the matrix does complicate that.

:shrug:

Reply:
Yeah, I could change it to something else in this PR if you want.